python/uds.py

```diff
@@ -477,7 +477,7 @@ def _isotp_rx_next(self, rx_data: bytes) -> ISOTP_FRAME_TYPE:
       assert self.rx_len < self.max_len, f"isotp - rx: invalid single frame length: {self.rx_len}"
       self.rx_dat = rx_data[1:1 + self.rx_len]
       self.rx_idx = 0
-      self.rx_done = True
+      self.rx_done = error_code != 0x78  # todo me
```

Comment:
needs to handle response pending. you get a single frame, then a first or single frame which is valid